docs/source/executing-operations/native-query-planner.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: Native Query Planner
+subtitle: Run the Rust-native query planner in GraphOS Router
+minVersion: 1.49.0
+---
+
+<PreviewFeature />
+
+Learn to run the GraphOS Router with the Rust-native query planner and improve your query planning performance and scalability.
+
+## Background about query planner implementations
+
+In v1.49.0 the router introduced a [query planner](/graphos/routing/about-router#query-planning) implemented natively in Rust. This native query planner improves the overall performance and resource utilization of query planning. It exists alongside the legacy Javascript implementation. and it will eventually replace the legacy implementation.
+
+As part of the effort to ensure correctness and stability of the new query planner, starting in v1.53.0 the router can enable both the new and legacy planners and run them in parallel to compare their results. After their comparison, the router discards the native query planner's results and uses only the legacy planner to execute requests.
```

Comment:
So this happens by _default_ since 1.53.0. We are not switching the default of running both side by side for this release.
```suggestion
As part of the effort to ensure correctness and stability of the new query planner, starting in v1.53.0 the router enables both the new and legacy planners and runs them in parallel to compare their results by default. After their comparison, the router discards the native query planner's results and uses only the legacy planner to execute requests.
```